src/components/AddItem.js

```diff
@@ -54,62 +67,64 @@ const AddItem = ({ token }) => {
   };
 
   return (
-    <div>
-      <h1>Add Item</h1>
-      <form onSubmit={handleSubmit}>
-        <label>
-          Item Name
-          <input
-            type="text"
-            placeholder="add your item here"
-            name="itemName"
-            required
-            value={formData.itemName}
+    <Box
+      display="flex"
+      justifyContent="center"
+      alignItems="center"
+      width={500}
+      height={300}
+      border={3}
+      borderColor="#d3d3d3"
+      borderRadius="borderRadius"
+    >
+      <form className="form-container" onSubmit={handleSubmit}>
```

Comment:
Can I get suggestions to the overall `form` element on line 80? I'm trying to figure out if there is a better way using a Material UI component.